components/userRegistration.js

```diff
@@ -44,6 +47,16 @@ const userRegistration = (slackController) => {
 
           testBot.team_info = team;
 
+          if (!isNewTeam) {
```

Comment:
Would checking if the team ID exists in the DB already before saving it (line 60, below) also solve the issue? That might be cleaner than deleting an existing entry and adding it again.

Reply:
@phoebe-lew yeah that also solve the issue, but my concern is if workspace information changed, it won't update the existing record in `botkit_team`.

Reply:
That's true, fair enough!